content/en/docs/reference/command-line-tools-reference/feature-gates/kubelet-separate-disk-gc.md

```diff
@@ -6,9 +6,13 @@ _build:
   render: false
 
 stages:
-  - stage: alpha 
```

Comment:
_nit_: Avoid change to this existing line.

Reply:
This line has surplus whitespaces that _should not be there_.